pkg/dumper/postgres/postgres.go

```diff
@@ -0,0 +1,27 @@
+package postgres
+
+import (
+	"database/sql"
+	"strings"
+
+	"github.com/hellofresh/klepto/pkg/dumper"
+	"github.com/hellofresh/klepto/pkg/reader"
+)
+
+type driver struct{}
+
+func (m *driver) IsSupported(dsn string) bool {
+	return strings.HasPrefix(strings.ToLower(dsn), "postgres://")
```

Comment:
This should be reflected in the docs. Also, how we identify mysql (perhaps we should add `mysql://` to that for consistency...)